sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -32,9 +32,8 @@
    <version ClassVersion="10" checksum="83320005"/>
   </class>
 
-  <class name="caf::SRSlice" ClassVersion="12">
-   <version ClassVersion="12" checksum="3783103871"/>
```

Comment:
I think the legacy versions are required for backwards compatibility?

Reply:
Agreed. Any version number that has been in `develop` should remain here.